recipes/tensorflow-metadata/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "tensorflow-metadata" %}
+{% set version = 0.30.0 %}
```

Missing quotes break the CI job
```suggestion
{% set version = "0.30.0" %}
```

---

whoops. thanks.